code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -385,6 +385,21 @@
 	new /obj/item/device/firing_pin/implant/loyalty(src)
 	new /obj/item/device/firing_pin/implant/loyalty(src)
 
+/obj/item/weapon/storage/box/firingpinsRD
+	name = "box of assorted firing pins"
+	desc = "A box of varied assortment of firing pins. Appears to have R&D stickers on all sides of the box. Also seems to have a smiley face sticker on the top of it."
+
+/obj/item/weapon/storage/box/firingpins/fill()
+	..()
+	new /obj/item/device/firing_pin(src)
+	new /obj/item/device/firing_pin(src)
+	new /obj/item/device/firing_pin/implant(src)

```

Comment:
The `/obj/item/device/firing_pin/implant` pins are useless ingame, as you wont be able to fire them.
Replace them with `/obj/item/device/firing_pin/access`

Reply:
Changes made.